timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -159,6 +170,7 @@ def _get_mlforecast_init_args(self, train_data: TimeSeriesDataFrame, model_param
 
         target_transforms = []
         differences = model_params.get("differences")
+        assert isinstance(differences, list)
```

Comment:
Can we either cast or add a comment describing this? I can imagine that some users will provide this as a `np.ndarray` - we probably don't want to break the model for them just to appease the type checker.

Reply:
converted to `Collection`